pkg/manifest/manifest.go

```diff
@@ -151,17 +156,78 @@ func (m *Manifest) Include(excludeIdentifier *string, includeTechPreview *bool,
 	return nil
 }
 
+// ManifestEnabled returns whether the manifest is enabled given the defined cluster capabilities.
```

Comment:
We should move `func checkResourceEnablement` just after `func (m *Manifest) Include` and the new methods can be inserted after that. As `checkResourceEnablement` only used in the include method. Just a nit pick, not something blocking this PR from merging